.pre-commit-config.yaml

```
@@ -29,27 +29,27 @@ repos:
     - id: ruff-check
       args: ["--fix", "--show-fixes"]
     - id: ruff-format
 
 - repo: https://github.com/adamchainz/blacken-docs
-  rev: "1.20.0"
+  rev: "fda77690955e9b63c6687d8806bafd56a526e45f"  # frozen: 1.20.0
   hooks:
     - id: blacken-docs
       additional_dependencies: [black==26.3.1]
 
 -   repo: https://github.com/pre-commit/mirrors-mypy
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)